bot/resources/tags/dict-get.md

```diff
@@ -0,0 +1,15 @@
+Often while using dictionaries in Python, you may run into `KeyErrors`. This error is raised when you try to access a key that isn't present in your dictionary.\
+While you can use a `try` and `except` block to catch the `KeyError`, Python also gives you some other neat ways to handle them.
+
+__**The `dict.get` method**__
+
+The [`dict.get`](https://docs.python.org/3/library/stdtypes.html#dict.get) method will return the value for the key if it exists, or None (or a default value that you specify) if the key doesn't exist. Hence it will _never raise_ a KeyError.
+```py
+>>> my_dict = {"foo": 1, "bar": 2}
+>>> print(my_dict.get("foobar"))
+None
+>>> print(my_dict.get("foobar", 3))    # here 3 is the default value to be returned, because the key doesn't exist
+3
```

Comment:
```suggestion
>>> print(my_dict.get("foobar", 3))  # Here, 3 is specified as the default value
3  # The default value is returned, since the key doesn't exist
```
Let's make sure that the comment follows PEP 8. I'd also like to see this split up in a way like the above, but I'll leave the details to you.

Reply:
I didn't know how to split the comment whilst also keeping to the REPL look (with the `>>>`), wouldn't a lone comment look a bit odd?

Reply:
Hmmm. This looks fine to me. It might also be worth considering putting the explanation outside of the code block, if you don't like putting the comment on the new line.